nengo/dists.py

```diff
@@ -309,11 +328,52 @@ class SubvectorLength(SqrtBeta):
     --------
     SqrtBeta
     """
+
     def __init__(self, dimensions, subdimensions=1):
         super(SubvectorLength, self).__init__(
             dimensions - subdimensions, subdimensions)
 
 
+class CosineSimilarity(SubvectorLength):
+    """Distribution of dot products between random unit vectors.
+
+    This is equivalent to the distribution of a single coefficient in a unit
+    vector (i.e. a single dimension of `UniformHypersphere(surface=True)`),
+    which can be obtained from `SubvectorLength` by symmetry.
+
+    This can be used to calculate an intercept `c = invcdf(1 - p)` such that
```

Comment:
Would it make sense to be [consistend with SciPy and use `ppf`](http://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.rv_continuous.ppf.html#scipy.stats.rv_continuous.ppf)?